integrators_test.go

```diff
@@ -0,0 +1,148 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/faiface/pixel"
+)
+
+func createParticle(pos pixel.Vec, nextPos pixel.Vec, speed pixel.Vec, prevDT float64, lifespan float64) Particle {
+	particleImage, err := loadPicture("assets/sprites/particle.png")
+	if err != nil {
+		panic(err)
+	}
+
+	particleSprite := pixel.NewSprite(
+		particleImage,
+		pixel.R(
+			particleImage.Bounds().Min.X,
+			particleImage.Bounds().Min.Y,
+			particleImage.Bounds().Max.X,
+			particleImage.Bounds().Max.Y,
+		))
+
+	p := Particle{
+		position:     pos,
+		nextPosition: nextPos,
+		speed:        speed,
+		prevDt:       prevDT,
+		sprite:       *particleSprite,
+		lifespan:     lifespan,
+		alive:        0.0,
+	}
+
+	return p
+}
+
+// TestP01 tests particle initialization
+func TestP01(t *testing.T) {
+	var (
+		pos           = pixel.V(0, 0)
+		nextPos       = pixel.V(0, 0)
+		speed         = pixel.V(0, 10)
+		prevDt        = 0.0
+		lifespan      = 10.0
+		ePosition     = pixel.V(0, 0)
+		eNextPosition = pixel.V(0, 0)
+		eSpeed        = pixel.V(0, 10)
+		eLifespan     = 10.0
+	)
+
+	p := createParticle(pos, nextPos, speed, prevDt, lifespan)
+
+	if p.position != pixel.V(0, 0) {
+		t.Errorf("Particle initialization: Expected position of %f got %f", ePosition, p.position)
+	}
+
+	if p.nextPosition != pixel.V(0, 0) {
+		t.Errorf(
+			"Particle initialization: Expected next position of %f got %f",
+			eNextPosition, p.nextPosition)
```

Comment:
Is there a better way in go to format this lines? sth like:
```go
t.Errorf(
    "Particle initialization: Expected next position of %f got %f",
    eNextPosition, p.nextPosition
)
```
If yes, than format all occurrence of this.